apps/auction2/components/bid_status/index.js

```diff
@@ -0,0 +1,23 @@
+import { default as React, PropTypes } from 'react';
+
+export default function BidStatus({ artwork }, _) {
+  const saleArtwork = artwork.get('sale_artwork')
+
+  let bidLabel
+  if (saleArtwork.counts.bidder_positions > 0) {
+    bidLabel = `<b>Current Bid:</b> (${saleArtwork.counts.bidder_positions} Bids)`
```

Comment:
I think you could use JSX here and avoid the `dangerouslySetInnerHTML` e.g. ` bidLabel = <b>Current Bid:</b>...`